ozaria/site/templates/play/play-level-view.pug

```diff
@@ -7,7 +7,7 @@ if view.showAds()
     script.
       (adsbygoogle = window.adsbygoogle || []).push({});
 
-//- include game-dev-goals.jade
+include game-dev-goals
```

Comment:
Was this the reason game dev levels were not working?

Reply:
Yeah. I wonder why it's showing up as a diff.

Reply:
Will rebase again and that should clear this up 👍